packages/block-editor/src/components/media-placeholder/README.md

```diff
@@ -27,6 +27,87 @@ const { MediaPlaceholder } = wp.editor;
 	}
 ```
 
+## Props
+
+### accept
```

Comment:
Should it be clarified what's the difference between `accept` and `allowedTypes`, if any?